src/Tizen.NUI.Components/Controls/RecyclerView/LinearLayoutManager.cs

```diff
@@ -29,6 +29,31 @@ namespace Tizen.NUI.Components
     public class LinearListLayoutManager : LayoutManager
```

Comment:
I'm a bit confused with the class name because it's different from the file name (= LinearLayoutManager).

Reply:
FlexibleView already has LinearLayoutManager and GridLayoutManager but they are not public and not used.
So, We will remove FlexibleView first and after that, will change LinearListLayoutManager to LinearLayoutManger.

Reply:
I got it. Thanks for answering :)